packages/code/src/Code/Code.styles.ts

```diff
@@ -98,6 +124,22 @@ export const singleLineCodeWrapperStyle = css`
   padding-bottom: ${(singleLineComponentHeight - lineHeight) / 2}px;
 `;
 
+export function getExpandableCodeWrapperStyle(
```

Comment:
Should there be a transition?

Reply:
I didn't see anything communicated about a transition on expansion/collapse. I've reached out to Rob to get his thoughts.

Reply:
Discussed with Rob and will be adding a 300-500ms animation on expansion/collapse here

Reply:
From what I've seen, we usually wait until a feature is fully supported before we use it, so I would hold off on using `auto`